src/mono/wasm/runtime/invoke-cs.ts

```diff
@@ -127,6 +127,7 @@ function bind_fn_0V(closure: BindingClosure) {
     (<any>closure) = null;
     return function bound_fn_0V() {
         const mark = startMeasure();
+        loaderHelpers.assert_runtime_running();
```

Comment:
If this assertion fails and throws will it leave a dangling measure that never stops measuring?

Reply:
it's NOOP unless enabled. When enabled it will create named timestamp in the browser profiler. Not big deal.

I'm rather thinking, should this call abort and kill the whole app ?

Reply:
Should any mono_assert call `Module.abort` ? Because C code assert does, right ?

Reply:
That sounds like a good idea in principle, I can't think of anything wrong with it